src/NzbDrone.Common.Test/DiskTests/DiskTransferServiceFixture.cs

```diff
@@ -0,0 +1,314 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Moq;
+using NUnit.Framework;
+using NzbDrone.Common.Disk;
+using NzbDrone.Test.Common;
+using FluentAssertions;
+
+namespace NzbDrone.Common.Test.DiskTests
+{
+    [TestFixture]
+    public class DiskTransferServiceFixture : TestBase<DiskTransferService>
```

Comment:
AFAICT, you don't have any tests for when copy/move fails because destination is readonly or doesn't exists.